src/components/Areas/Areas.js

```diff
@@ -0,0 +1,18 @@
+function Areas({ bodyParts, side, handleClick }) {
```

Comment:
Nawiązując do zmiany struktury danych, to obsługa `side` uprości się i będzie można tylko po unikalnym `id` określać która część ciała się zmieniła.